src/modules/rc_update/rc_update.h

```diff
@@ -90,8 +90,6 @@ class RCUpdate : public ModuleBase<RCUpdate>, public ModuleParams, public px4::W
 
 	int print_status() override;
 
-private:
```

Comment:
You can use `friend class xy;`, and for gtest `FRIEND_TEST(TestCaseName, TestName);`

Reply:
I thought about that but did not go down this route because I want to strictly avoid dependency of the tested class on the tests and gtest.
Other people tried this before: https://discuss.px4.io/t/useing-friend-test-for-unit-functional-gtests/16058

My goal is to allow this test to verify I'm not breaking anything and then amend the structure of `RCUpdate` such that it's not necessary anymore to access private internals in order to test functionality. The `private:` then goes back where it was.

In theory, I could use `protected:` and move all tests into a dummy class that inherits from `RCUpdate`. I didn't do this because everything gets less readable just to protect the members of `RCUpdate` while the class is only ever instantiated as its own module.